src/logic/Locations.js

```diff
@@ -90,18 +90,18 @@ class Locations {
         _.unset(this.locations, [area, location]);
     }
 
-    splitLocationName(name) {
+    static splitLocationName(name) {
         const locationElements = name.split('-');
```

Comment:
was about to comment why not use `split('-',1)` but apparently javascript is badly designed so that doesn't work
If you want to be accurate to the randomizer though, it splits at ' - ' (including whitespace). Also when joining the string together I would personally keep the remaining `-` separators

Reply:
Good catch, that should have been written like that in the first place (with rejoining). It was minor so it slipped through everything I looked at (and also only appears once iirc)